RELEASE_NOTES.rst

```diff
@@ -1,4 +1,139 @@
+        \      /\        | \         |    |
+        |     |  \       |  \        |     \
+        |     |   |     /    |      /\      |
+        |    /    |    |     |     /  |      \
+       /_____}   /_____}    /______}  {_______}
+
+^ Fabian, gimme something better than this :P
+                                                                                                 
+
+For this release, we have had 45 developers from around the world contributing
+over 148 commits in the last 8 weeks.
+
+Improving the API available for bear writers is one of the areas we've focussed
+on for this release, with several new and exciting features. General performance
+has also been improved heavily with some major changes under the hood. The
+documentation has also been worked on, with an emphasis on user-friendliness
+and design.
+
+There have also been major internal changes in preparation for the complete
+decentralization of bears, which would allow the installation of individual
+bears.
+
+Below are some of the important changes introduced for this release:
+
+**New Features**
+
+- coala now supports syntax highlighting in results!
+
+- Questions are now printed in color; this will improve visibility when a lot
+  of text is written to the screen.
+
+- ``coala-json`` now supports ``--show-bears`` and ``--filter-by-language``
+
+- Added a ``--show-capabilities`` flag that displays the types of issues coala
+  can detect and fix for a particular language.
+
+- Display the line number when a line is missing; this could happen if a bear
+  that had run previously overwrites it.
+
+**For Bear Writers**
+
+- Bears now have a new ``REQUIREMENTS`` attribute which will be used to
+  automatically resolve bear dependencies. This includes:
+
+  + Native requirements (from package managers such as ``apt-get``, ``dnf``, ``pacman``, ...)
+  + Conda requirements
+  + Python requirements through ``pip``
+  + ``go`` requirements
+  + Ruby requirements through ``gem``
+  + NodeJS requirements through ``npm``
+  + RScript requirements
+  + Julia requirements
+
+- Language independent documentation parsing routines: these can be used to
+  make bears for linting documentation without having to worry about the
+  language.
+
+- ``coalang`` now supports C, C++, CSS, Java, Python3 and Vala.
+
+- A new `ASCIINEMA_URL` attribute has been added to bears. This should
+  contain an URL to an asciinema video displaying the bear's capabilities in action.
+
+- Bear results may now have a ``confidence`` parameter: this is supposed to
+  quantify the confidence, on a scale of 1 to 100, the bear has when flagging results.
+
+- A ``deprecate_settings`` decorator has been created to deprecate old,
+  unsupported bear parameters. Please see
+  `here <https://github.com/coala-analyzer/coala/blob/fa8fe22562277762fd73ab3761ad1ec33263839a/coalib/bearlib/__init__.py#L15>`_
+  for an example usage.
+
+- A new ``CAN_FIX`` value ``Code Simplification`` has been introduced.
```

Comment:
This is a very minor point actually. Debatable whether it should remain in the changelog